Tests/DatadogTests/Datadog/Mocks/RUMDataModelMocks.swift

```diff
@@ -4,6 +4,7 @@
  * Copyright 2019-Present Datadog, Inc.
  */
 
+import TestUtilities
```

Comment:
One thing I just remembered. The idea was to use `@testable import TestUtilities` mainly to avoid changing all the access modifiers inside the package. In the end of the day it actually turns all internal to public automatically.

I'm wondering why you didn't have to change anything inside TestUtilities and it still worked 🤔

Reply:
I did change all accessor to `public` :) It was only few files in the package.